Maths/SimpsonIntegration.js

```diff
@@ -64,4 +64,51 @@ function integralEvaluation (N, a, b, func) {
   return result
 }
 
-export { integralEvaluation }
+function integralEvaluationRefractored (N, a, b, func) {
```

Comment:
I think that we're supposed to refactor the original function rather than creating a refactored copy since that's how we'd do it in a real scenario. Your changes will still be clearly visible in the Github comparison view.  Also, this refactored copy is not checked by the unit tests.